room/compiler-processing/src/main/java/androidx/room/compiler/processing/XProcessingEnv.kt

```diff
@@ -128,11 +134,19 @@ interface XProcessingEnv {
         KSP
     }
 
+    enum class Language {
+        JAVA,
+        KOTLIN
+    }
+
```

Comment:
sorry i forgot to put comments here but it would be good to add comments here.

```suggestion
    /**
     * The target language for XProcessingEnv.
     * This value controls certain logic when processing code, see the documentation for [XProcessingEnv.targetLanguage] for details.
     */
    enum class Language {
        JAVA,
        KOTLIN
    }

```

Reply:
Indeed, done.